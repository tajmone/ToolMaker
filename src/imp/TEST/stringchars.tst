%%SET foo("!@#$%^&*()-_=+[{]}'`~;:/?.>,<\|%%""%%SET<=>=")
%%(foo)
