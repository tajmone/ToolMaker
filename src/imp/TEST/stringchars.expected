########## stringchars ##########
%%SET foo("!@#$%^&*()-_=+[{]}'`~;:/?.>,<\|%%""%%SET<=>=")
%%(foo)
########## ../imp stringchars.tst >stringchars.tst.tmp; cat stringchars.tst.tmp; rm stringchars.tst.tmp ##########
!@#$%^&*()-_=+[{]}'`~;:/?.>,<\|%%"%%SET<=>=
